ansible_collections/ds/ansible_ds/playbooks/dscreate.yml

```diff
@@ -0,0 +1,47 @@
+- name: Playbook to create a new instance if it does not alreadfy exist
+  hosts: localhost
+  collections:
+    - ds.ansible_ds
+
+  roles:
+  - role: ansible_collections/ds/ansible_ds/playbooks/roles/install_dspkg
+    state: present
+
+  vars:
+    instdict:  "{{
+        {
+            instance:
+              {
+                'state': 'present',
+                'port': port,
+                'secure_port': secure_port,
+                'root_password': rootpw,
+                'backends': bedict,
+              }
+        }
+    }}"
```

Comment:
I think it'll be more 'Ansible way' if we do like FreeIPA had done - https://github.com/freeipa/ansible-freeipa/blob/master/playbooks/install-server.yml :

```
---
- name: Playbook to configure IPA servers
  hosts: ipaserver
  become: true

  roles:
  - role: ipaserver
    state: present
```

And when the administrator would like to delete the instance, he will say - 'state: absent' (it can be a task for another PR, probably).

So something like:

```
---
- name: Playbook to configure DS servers
  hosts: dsserver
  become: true

  roles:
  - role: dsserver
    state: present
```

Ideally, it should also be possible to specify some init settings for the instance (may be the same list we have in 'dscreate create-template' command):

```
---
- name: Playbook to configure DS servers
  hosts: dsserver
  become: true
  
  vars:
    port: 389
    secure_port: 636
    # etc.
  
  roles:
  - role: dsserver
    state: present
```

So if it's already installed, it'll make sure that everything is correctly set in the instance.
And more fine-grained updates can be achieved with 'dsupdate' role/task.
I am not sure 'instdict' with the big JSON approach is an Ansible correct one. (we may ask Rich Megginson for guidance here, probably) 
 
What do you think?